app/Annotation.php

```diff
@@ -60,7 +60,8 @@ public function scopeVisibleFor($query, User $user)
                 ->join($ownerTable, $ownerKeyName, '=', $foreignKeyName)
                 ->join('project_volume', 'project_volume.volume_id', '=', "{$ownerTable}.volume_id")
                 ->whereIn('project_volume.project_id', function ($query) use ($user) {
-                    $query->select('project_id')
+                    $query
+                        ->select('project_id')
                         ->from('project_user')
                         ->where('user_id', $user->id);
```

Comment:
I think instances like these are no problem with the CS fixer. Only lines that start a closure or array which is not closed on the same line are an issue. So I would leave lines like this one untouched (as line 53 above is untouched).